### DISCUSSION TITLE
SO 2.4 Failed to compile

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#13276] [Unsupported Versions]
Hello,

I have updated my system and getting this error

local:
    Data failed to compile:
----------
    No matching sls found for 'mysql' in env 'base'
----------
    No matching sls found for 'soctopus' in env 'base'
----------
    No matching sls found for 'playbook' in env 'base'

**Comment by cm-ops** [comment]
Do you have a local copy of your `top.sls` in `/opt/so/saltstack/local/salt/`?
